articles/client-platforms/angularjs.md

```diff
@@ -13,14 +13,16 @@ tags:
   - quickstart
 snippets:
   dependencies: client-platforms/angularjs/dependencies
-  setup: client-platforms/angularjs/setup
-  use: client-platforms/angularjs/use
```

Comment:
@christiannwamba So the `dependencies`, `setup`, and `use` scripts are required. They are used for some of our SEO pages on auth0.com. You can change the content of the script, but not the name.